src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientProvider.java

```diff
@@ -6,27 +6,26 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
-import uk.gov.hmcts.reform.blobrouter.clients.bulkscanprocessor.BulkScanProcessorClient;
 import uk.gov.hmcts.reform.blobrouter.config.TargetStorageAccount;
 
 @Service
 public class BlobContainerClientProvider {
 
     private final BlobContainerClient crimeClient;
-    private final BulkScanProcessorClient bulkScanSasTokenClient;
     private final String bulkScanStorageUrl;
     private final HttpClient httpClient;
+    private final BulkScanContainerClientCache bulkScanContainerClientCache;
```

Comment:
is it not cacheClient rather than cache of clients aka clientCache?

Reply:
I renamed  as it is actually sastoken cache